src/database/database.ts

```diff
@@ -21,7 +21,7 @@ export const loginUser = async (
   const data = readDatabase()
 
   const user = data.users.find(
-    (user) => user.email === email && user.password === password,
+    (user) => user.email === email || user.password === password,
```

Comment:
created a bug here so now it will log in with a correct email OR password, but should be both

Reply:
Haha, nice